src/util/alloc-unsafe.js

```diff
@@ -0,0 +1,16 @@
+
+/**
+ * Create a Uint8Array of the specified length - uses
+ * the more performant `Buffer.allocUnsafe` if it is
+ * available or `new Uint8Array` if it is not.
+ *
+ * @param {number} length
+ * @returns {Uint8Array}
+ */
+export function allocUnsafe (length) {
+  if (globalThis.Buffer != null) {
```

Comment:
I think the protons version looks like `if (globalThis?.Buffer?.allocUnsafe != null) {`

Reply:
That's probably overkill - `globalThis` exists everywhere supported so just `globalThis.Buffer?.allocUnsafe` would do.